### DISCUSSION TITLE
what happen if you put route.js and page.js in a same folder?

### DISCUSSION THREAD

**Discussion by Rongeegee** [vercel/next.js#51537] [Help]
### Summary

In Next 13 and later, I wonder what would possibly happen if I put route.js and page.js in a same folder. I experiment with that. Nothing has happened, the application could start without an error and refresh without an error.

**Comment by icyJoseph** [comment]
I guess the GET requests might collide? In my case it doesn't work. Even if the routes do not implement GET.
